CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ option(VW_BOOST_MATH_SYS_DEP "Override using the submodule for boost math depend
 option(VW_ZLIB_SYS_DEP "Override using the submodule for zlib dependency. Instead will use find_package" ON)
 option(VW_GTEST_SYS_DEP "Override using fetch package for gtest dependency. Instead will use find_package" OFF)
 option(VW_BUILD_VW_C_WRAPPER "Enable building the c_wrapper project" ON)
+option(BUILD_CSV "Build csv parser" OFF)
```

Comment:
I believe it's generally preferred to name your options like `VW_BUILD_CSV` to avoid duplicate names with other cmake projects

Reply:
+1 to this

Reply:
Fixed, thanks for reviewing!